cmd/whitesourceExecuteScan.go

```diff
@@ -166,6 +177,9 @@ func triggerWhitesourceScan(cmd *command.Command, config *ScanOptions) error {
 // Executes a scan with the Whitesource Unified Agent
 // returns stdout buffer of the unified agent for token extraction in case of multi-module gradle project
 func executeUAScan(config *ScanOptions, cmd *command.Command) error {
+	log.Entry().Info("Starting whitesource scan with parameters: ", "java", "-jar", config.AgentFileName,
+		"-d", ".", "-c", config.ConfigFilePath, "-apiKey", config.OrgToken, "-userKey", config.UserToken,
+		"-project", config.ProjectName, "-product", config.ProductName, "-productVersion", config.ProductVersion)
```

Comment:
This output should be redundant (except for the "Starting whitesource scan" part), since `RunExecutable` already outputs the same to the log.